metaforce.gemspec

```diff
@@ -31,7 +31,7 @@ EOL
   s.add_dependency 'faraday'
 
   s.add_development_dependency 'rake'
-  s.add_development_dependency 'rspec'
+  s.add_development_dependency 'rspec', '~> 2.0'
```

Comment:
This is a pretty loose restriction. Did you check that specs still pass with '2.99'? In other cases I found that introduced some of the same upgrade issues as '3.0'.